driver/setup.py

```diff
@@ -6,7 +6,7 @@
 setup_requirements: List[str] = []
 
 requirements = [
-    "util",
+    "pace-util",
```

Comment:
why is it necessary to have `pace-util` in the setup.py files?

Reply:
Because that's the name of the package, and `pace-driver` doesn't run without it.